### ISSUE TITLE
problem about retraining the model

### ISSUE THREAD

**Issue by zhzxlcc** [FilippoAleotti/DWARF-Tensorflow#4]
Excuse me, I want to retrain the model in the Flyingthings3D dataset and have set the parameters according to the paper. But I find a strange thing: step=85000 and 475000, the optical flow epe error is reduced from 12% to 7%. The disparity and disparity change epe error are still at 42%  and didn't fall down. What's the problem behind it?
Besides, I wonder to know how to use the knowledge distillation for better performance.
Thanks very much.

**Comment by FilippoAleotti** [comment]
Hi,
The network requires a long training time (especially in the synthetic domain). However, what do you mean by 42% epe error? If the network has a 42 epe error, there is something wrong (maybe in data or in training setup).
About distillation, I used it only in the real domain

**Comment by zhzxlcc** [comment]
Thanks for you reply. 
Yes, the network has a 42 epe error with respect to the disparity and disparity change. Since  ./configurations/sceneflow.json lacks the keys and parameters for training mode, I have set up the configuration on flyingthings according to the paper. Maybe something was missing. Could you please provide the training setup for sceneflow.json? Thanks a lot!

**Comment by zhzxlcc** [comment]
Sorry to bother you again~
With the help of you, I have almost achieved the same performance on Flyingthings3D. However, when trying to finetune the model on KITTI just with the provided gt, I got the results as follows:F1-27%,D1-6%,D2-9%(Noc);F1-36%,D1-6%,D2-15%(All).
I think the flow outliers are far from the error 18.53% in the paper obviously. Is the ablation study of KITTI in terms of Noc or All? And I wonder to know if you employed cross-validation or prolong the training process or just finetune the model with the kitti.json?  Some suggestions are helpeful~
Thanks very much!

**Comment by FilippoAleotti** [comment]
Hi,
The results are about OCC validation of the model fine-tuned on KITTI

**Comment by zhzxlcc** [comment]
Thanks a lot! I still have problems about the performance on KITTI dataset. The error is far from the normal value. I'll check it. Thanks for your attention to this matter.

**Comment by FilippoAleotti** [comment]
Hi,
Are you using kitti_160 split right? And NN as interpolation method in filter_settings?

**Comment by zhzxlcc** [comment]
In fact, I create txt for kitti_170 split according to your kitti_160.txt and use the last 30 pictures as validation. And do you mean I should modify "NN as interpolation" in "filters_setting{method :NN}" of the kitti.json in the configurations?
I didn't modify the parameters in the kitti.json before. After setting the filters_setting:method as NN, I got the following results with respect to OCC: F1-33%,D1-5%,D2-13%. Maybe something wrong with corr2d when run "python ops.py“. It is strange that only flow outliers is abnormal.
Thanks a lot !

**Comment by FilippoAleotti** [comment]
Are you using the evaluate_partial_scene_flow script inside external_packages/kitti_test_suite for evaluation (make sure to change it if you are using 170 images for training, and not 160)?

**Comment by zhzxlcc** [comment]
Thanks for your attention. For valuation, I use the KITTI official code implemented in MATLAB. By the time, I got the OCC results for 30 valuation set  as follows: F1-22%,D1-4%,D2-9%(All). I think they are closer to the results only using KITTI gt in your awesome paper~ Would you consider it is the reasonable result?
I am very grateful for your guidance.
